lib/lita/handlers/ai.rb

```diff
@@ -11,6 +11,7 @@ def chat(payload)
         return unless chatting?(payload[:message])
         message = extract_aliases(payload[:message])
         reply = self.class.cleverbot.think(message.body)
+        reply = reply.to_s.split('|').map { |s| s.length == 4 ? [s.hex].pack("U") : s }.join('') if reply.to_s[0] == '|'
```

Comment:
http://d.pr/i/QW3G

I guess the pattern may be `/\|[0-9A-F]{4}/`
so that better way may be:
```ruby
reply.gsub!(/\|([0-9A-F]{4})/) { ["#{$1}".hex].pack("U") }
```
I tried Chinese, Japanese, Korea, Arabic, Latin, France, Germany are 4 characters with prefix `|`
This way may handler most languages of world and would not affect original function (English).
How about you? :p